### DISCUSSION TITLE
Unable to figure out how to do config like this.

### DISCUSSION THREAD

**Discussion by semi710** [fastfetch-cli/fastfetch#556] [Q&A] [answered]
![image](https://github.com/fastfetch-cli/fastfetch/assets/60490474/678d269c-4b3f-4f56-adb5-ec58ac759b56)

**Comment by CarterLi** [comment]
You may start it with <https://github.com/fastfetch-cli/fastfetch/blob/dev/presets/examples/7.jsonc>

The result is:

<img width="747" alt="image" src="https://github.com/fastfetch-cli/fastfetch/assets/6134068/078db358-5fbc-4e04-ab51-ddafe4a470b3">

The top and bottom borders can be displayed with `Custom` module. Like this one: https://github.com/fastfetch-cli/fastfetch/blob/dev/presets/examples/2.jsonc#L18

<img width="747" alt="image" src="https://github.com/fastfetch-cli/fastfetch/assets/6134068/7fe9e049-61c1-48e1-b2d3-283cdc17b7ce">

The left border can be displayed with custom keys. eg:

```jsonc
{
    "type": "kernel",
    "key": "│ ├",
    "keyColor": "yellow"
}
```

**Comment by CarterLi** [comment, marked as answer]
[Done](https://github.com/fastfetch-cli/fastfetch/blob/dev/presets/examples/10.jsonc)

<img width="747" alt="image" src="https://github.com/fastfetch-cli/fastfetch/assets/6134068/cfbec43e-94a2-4e95-a6f9-88bda03966af">

**Reply by semi710** [reply]
This is what exactly i was looking ......... was close to this after reading docs but not exactly to this.

thanks mate you acheived this very simply and easily i was doing it in a much complex way XD
